kolibri/core/assets/src/views/k-navbar/items.styl

```diff
@@ -17,7 +17,6 @@
     border-radius: 0
     font-size: 14px
     text-decoration: none
-    color: $core-text-annotation
```

Comment:
Originally used that color to match the Material tabs spec, but don't mind this change.

Reply:
oh whoops - for some reason I thought this was a mistake. Will revert